quinn-udp/src/windows.rs

```diff
@@ -137,6 +132,25 @@ impl UdpSocketState {
         };
         Ok(1)
     }
+
+    /// The maximum amount of segments which can be transmitted if a platform
+    /// supports Generic Send Offload (GSO).
+    ///
+    /// This is 1 if the platform doesn't support GSO. Subject to change if errors are detected
```

Comment:
The "Subject to change" sentence doesn't really make sense in this context, I think?

Reply:
The intention is to describe the behavior of the platform-agnostic interface; the goal of quinn-udp is for the caller to not have to account for the quirks of any one platform. Portable code shouldn't assume this returns a constant, so the docs say as much.